packages/firestore/CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Unreleased
+- [fixed] Fixed a performance regression introduced in 1.7.0. (#2620)
```

Comment:
The public is largely unaware of the Firestore package's version (and our public release notes never mention them). I suggest tying this to "Firebase 7.3.0" somehow. You may also want to tie this to limitToLast to give users more of an anchor on exactly which version this was.

Maybe something like this:

> Fixed a performance regression introduced by the addition of `Query.limitToLast(n: number)` in Firestore 1.7.0 (Firebase 7.3.0) (#2620).

?

Reply:
Done.